### PR TITLE
Add docstring for the user_loader callback.

### PR DESCRIPTION
**Pull request by alandawkins**
Though this is not that significant, but I think this is necessary for 2 reasons: 1. it's hard to find clear answer for both why and how to correct this, especially for newbie; 2. this is my first pull request for a popular and great project, it matters! ;)

### CODE REVIEW

**Comment by alandawkins** [comment]
Oh, this issue is documented in README, but for people like me who get it from some books, they may not read the doc again, but search for solutions directly instead. :)

File: flask_login/login_manager.py [hunk 1]
```diff
@@ -299,7 +311,8 @@ def reload_user(self, user=None):
                     raise Exception(
                         "No user_loader has been installed for this "
                         "LoginManager. Add one with the "
-                        "'LoginManager.user_loader' decorator.")
+                        "'LoginManager.user_loader' decorator."
+                        "Refer to docstring for more information.")
```
Comment (alanhamlett) [inline flask_login/login_manager.py]: Let's put a link to the docs instead.

File: flask_login/login_manager.py [hunk 2]
```diff
@@ -288,6 +288,18 @@ def needs_refresh(self):
         return redirect(redirect_url)
 
     def reload_user(self, user=None):
+        '''
```
Comment (alanhamlett) [inline flask_login/login_manager.py]: Adding a docstring is good, but it should follow the same format as other docstrings with a one sentence description, and more info following.

**Comment by alandawkins** [comment]
I don't know what does it mean to put a link into exception message string, newbie to python. :(

**Comment by alanhamlett** [comment]
> what does it mean to put a link into exception message string
Just the same as any other text, but something like:

```python
raise Exception(
                          "No user_loader has been installed for this "
                          "LoginManager. Add one with the "
                          "'LoginManager.user_loader' decorator. "
                          "https://flask-login.readthedocs.io/en/latest/#how-it-works")
```

**Pull request merged**